core/src/main/java/org/frankframework/management/bus/endpoints/ExecuteJdbcQuery.java

```diff
@@ -55,7 +55,7 @@ public enum ResultType {
 	}
 
 	@ActionSelector(BusAction.GET)
-	@RolesAllowed({"IbisObserver", "IbisDataAdmin", "IbisAdmin", "IbisTester"})
+	@RolesAllowed("IbisTester")
```

Comment:
Iedereen mag de jdbc info zien toch, of is dit iets anders?

Reply:
Je heb BrowseJdbcTable en ExecuteJdbcQuery, enige wat ik kan bedenken is dat je hier de datasources terug krijgt zodat je daar een query naar kan sturen

Reply:
Observer access